commands/comment.go

```diff
@@ -58,7 +58,7 @@ func checkCommentLocation(repo repository.Repo, commit, file string, line uint)
 		return err
 	}
 	lines := strings.Split(contents, "\n")
-	if line >= uint(len(lines)) {
+	if line < 1 || line > uint(len(lines)) {
```

Comment:
First off, good catch on the upper-bound checking being wrong; it should use a > rather than a >=.

The lowerbound, however, needs to be a 0 rather than a 1, as leaving off the "-l" flag (i.e. commenting on the entire file) will result in the `line` variable having a value of 0, and that's a valid use case.

That being said, thanks for finding this issue and fixing it.